### PR TITLE
Update cucumber

### PR DESCRIPTION
**Pull request by rap1ds**
Merge after #1702 

- [X] Update Cucumber
- [X] Update our custom Sharetribe Formatter

### CODE REVIEW

File: features/support/formatter.rb [hunk 1]
```diff
@@ -1,13 +1,9 @@
 # encoding: utf-8
 
-require 'cucumber/formatter/console'
-require 'cucumber/formatter/io'
```
Comment (rap1ds) [inline features/support/formatter.rb]: These were duplicated

File: features/support/formatter.rb [hunk 2]
```diff
@@ -31,7 +27,7 @@ class Sharetribe
       attr_reader :runtime
 
       def initialize(runtime, path_or_io, options)
-        @runtime, @io, @options = runtime, ensure_io(path_or_io, "sharetribe"), options
```
Comment (rap1ds) [inline features/support/formatter.rb]: `ensure_io` has changed so that it only takes 1 argument

File: features/support/formatter.rb [hunk 3]
```diff
@@ -1,13 +1,9 @@
 # encoding: utf-8
 
-require 'cucumber/formatter/console'
-require 'cucumber/formatter/io'
-
-
 require 'fileutils'
 require 'cucumber/formatter/console'
 require 'cucumber/formatter/io'
-require 'gherkin/formatter/escaping'
```
Comment (rap1ds) [inline features/support/formatter.rb]: namespace changed

**Comment by Gnito** [comment]
Looks good too

**Pull request merged**